css/modules/styles.css

```diff
@@ -3874,12 +3644,20 @@ span#usertab_text {
 }
 
 #virtualclassCont.congrea.tr_available .handRaise {
-  pointer-events: visible !important;
+  pointer-events: visible;
   opacity: 1;
 }
 
 #virtualclassPreCheck #preCheckcontainer .modal-dialog .modal-content {
-  padding: 0 !important;
+  padding: 0;
+  max-height: unset;
+  box-shadow: 0 2px 10px 0 rgba(32,33,36,0.28) !important;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)